docs/UserGuide.md

```diff
@@ -295,11 +298,10 @@ Expected output:
 ### 7.7 Forming focus groups
 
 #### 7.7.1 Forming focus groups manually : `group`
-Creates a group with people corresponding to the selected IDs.
+Creates a group with people corresponding to the selected IDs. The group will have name `GROUP_NAME`.
```

Comment:
Would it make more sense to write "The group will have name GROUP_NAME" after line 303 where u specify where GROUP_NAME is in the command format?

Reply:
The reason why I shifted it in front was to conform to the format 

1. What the command does
2. The format of the command
3. Valid values that the command can take in
4. Example usages
5. _Optional Screenshot_.

That being said, I can write it in a clearer way.

Reply:
In this case I removed the line in question as it is self-evident that the group's name should be GROUP_NAME I think. Thanks for pointing it out.